xknx/devices/light.py

```diff
@@ -267,6 +267,11 @@ def _iter_remote_values(self) -> Iterator["RemoteValue"]:
             yield color.switch
             yield color.brightness
 
+    @property
+    def unique_id(self) -> Optional[str]:
+        """Return unique id for this device."""
+        return f"{self.switch.group_address}"
```

Comment:
I Just wanted to point out that the switch group address for light might be none when the individual colours feature is used. Perhaps we could use the individual colour addresses when switch.group_address is none?

Something like this:
```
if self.switch.group_address is not None:
    return f"{self.switch.group_address}"
else:
    return f"{self.red.switch.group_address}_{self.green.switch.group_address}_{self.blue.switch.group_address}_{self.white.switch.group_address}"
```